src/lightning/data/streaming/constants.py

```diff
@@ -57,3 +57,4 @@
 _NUMPY_DTYPES_MAPPING = {i: np.dtype(v) for i, v in enumerate(_NUMPY_SCTYPES)}
 
 _TIME_FORMAT = "%Y-%m-%d_%H-%M-%S.%fZ"
+_IS_IN_STUDIO = os.getenv("LIGHTNING_CLOUD_PROJECT_ID", None) and os.getenv("LIGHTNING_CLUSTER_ID", None)
```

Comment:
This should probably be made a boolean
```suggestion
_IS_IN_STUDIO = "LIGHTNING_CLOUD_PROJECT_ID" in os.environ and "LIGHTNING_CLUSTER_ID" in os.environ
```
or if these env variables can be empty, then perhaps:
```suggestion
_IS_IN_STUDIO = bool(os.getenv("LIGHTNING_CLOUD_PROJECT_ID", None)) and bool(os.getenv("LIGHTNING_CLUSTER_ID", None))
```